pod/perldelta.pod

```diff
@@ -476,7 +476,50 @@ manager will later use a regex to expand these into links.
 
 =item *
 
-XXX
+Lexical subs now have a new stub in the pad for each recursive call
+into the containing function.  This fixed two problems:
```

Comment:
Not sure about tenses in English. Is it "This fixed" or "This fixes"?

Reply:
s/fixed/fixes/